purchase_mass_mail/models/mail_compose_message.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    def send_mail(self, **kwargs):
+        super().send_mail(**kwargs)
+        context = self.env.context
+        if context["active_model"] == "purchase.order" and self._context.get(
```

Comment:
```suggestion
        if context.get("active_model") == "purchase.order" and context.get(
```

This is just to avoid a possible `KeyError` in case `active_model` doesn't exist in the dict.